features/admin/lib/src/ui/trash_screen.dart

```diff
@@ -42,12 +42,15 @@ class _TrashScreenState extends State<TrashScreen> {
                       onBackPress: () {
                         context.goNamed('admin');
                       },
-                      onUpdate: (String comment, String status, bool isVisible,
+                      onUpdate: (String name,
+                          String comment,
+                          String status,
+                          bool isVisible,
                           List<dio.MultipartFile> officerImages) {
                         context.read<TrashBloc>().add(UpdateReport(
                               id: state.trashReport.id,
                               refId: state.trashReport.refId,
-                              name: state.trashReport.name,
+                              name: name,
```

Comment:
I believe we can simplify cases like this by leveraging OpenAPI DTO builders. While the current implementation is acceptable for now, let's discuss these scenarios during our weekly meetings. Exploring the use of OpenAPI DTO builders may offer a more streamlined and maintainable solution for similar cases in the future.